internal/tools/scripts/schema-diff/yaml/schema-diff.j2

```diff
@@ -0,0 +1,18 @@
+{{- template.set_file_name("schema-diff.yaml") }}
```

Comment:
For clarity -

We're no longer doing renames *within* a metric or splits at this point right?

Previously defined diffs are preserved and we just don't allow those changes going forward?

Reply:
weaver.yaml filters changes to `renames` only. During code review we have to ensure things like scoped renames, splits or other not supported transformation are added as `uncategorized` instead of `renamed`. This is not new - we have to check this today as well.
 
So the short answer is yes,  this is only for proper renames.